manual/README.md

```diff
@@ -134,14 +134,21 @@ let f x = x
 \end{caml_example*}
 ```
 
-The `{verbatim}` or `{toplevel}` argument of the environment corresponds
-to the the mode of the example, two modes are available `toplevel` and
-`verbatim`.
+```latex
+\begin{caml_example*}{signature}
+val none : 'a option
+\end{caml_example*}
+```
```

Comment:
One last change: I think it is better to place this example after the description of the signature mode,


>If you want to declare a signature instead of ocaml code,
>you must use the `{signature}` argument to the `caml_example` environment:
>```latex
> \begin{caml_example*}{signature} 
> ...
>\end{caml_example*}
>```